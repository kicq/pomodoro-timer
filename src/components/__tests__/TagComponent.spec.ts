import TagComponentVue from '@/components/TagComponent.vue'
import { mount } from '@vue/test-utils'

const textSlot = 'My text'

describe('TagComponent mount', () => {
  
  const wrapper = mount(TagComponentVue, { slots: { default: textSlot }})

  it('TagComponent render truthy text ', () => {
    expect(wrapper.text()).toContain(textSlot)
  })

  it('TagComponent don\'t any classes ', () => {
    expect(wrapper.classes().length).toEqual(0)
  })

  it('should match the snapshot ', () => {
    expect(wrapper.html()).toMatchSnapshot()
  })
})