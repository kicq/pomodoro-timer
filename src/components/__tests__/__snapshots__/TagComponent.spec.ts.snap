// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TagComponent mount should match the snapshot  1`] = `<button>My text</button>`;
